docs/source/index.mdx

```diff
@@ -4,13 +4,14 @@ title: Introduction to Apollo Server
 
 **Apollo Server is an [open-source](https://github.com/apollographql/apollo-server), spec-compliant GraphQL server** that's compatible with any GraphQL client, including [Apollo Client](https://www.apollographql.com/docs/react). It's the best way to build a production-ready, self-documenting GraphQL API that can use data from any source.
 
-![diagram](./images/index-diagram.svg)
```

Comment:
Should we delete the old diagram?

Reply:
Oops, yes we should! I'll make a new PR —thanks for catching that!